hbase-server/src/test/java/org/apache/hadoop/hbase/tool/TestBulkLoadHFiles.java

```diff
@@ -623,6 +643,17 @@ private int verifyHFile(Path p) throws IOException {
     return count;
   }
 
+  private void verifyHFileCreateTimeTS(Path p) throws IOException {
+    Configuration conf = util.getConfiguration();
+    HFile.Reader reader =
+      HFile.createReader(p.getFileSystem(conf), p, new CacheConfig(conf), true, conf);
+    final HFileInfo hFileInfo = reader.getHFileInfo();
+    final long fileCreateTime = hFileInfo.getHFileContext().getFileCreateTime();
+    reader.close();
+    assertTrue(fileCreateTime != 0);
```

Comment:
Use hamcrest `assertThat(fileCreateTime, greaterThan(0));`

Reply:
thanks,I have modified it according to your suggestions. Please tell me if there are any shortcomings